modules/persistent-storage-csi-migration-overview.adoc

```diff
@@ -13,6 +13,8 @@ The following drivers are supported:
 
 * OpenStack Cinder
 
+* Google Compute Engine Persistent Disk (GCE PD) (in-tree) and Google Cloud Platform (GCP) PD (CSI)
```

Comment:
I suggest splitting these items into separate lines and/or considering either not using the product abbreviations or styling the types in something other than parentheses. (This line's hard to follow as-is.)